README.md

```diff
@@ -18,19 +23,28 @@ Starting
 --------
 
 Check out the lemur-docker and lemur repos:
-
-    git clone git@github.com:Netflix/lemur-docker.git
-    cd lemur-docker
-    git clone git@github.com:Netflix/lemur.git
+```bash
+git clone git@github.com:Netflix/lemur-docker.git
+cd lemur-docker
+git clone git@github.com:Netflix/lemur.git lemur-build-docker/lemur
+```
+
+Copy env files and edit if needed:
```

Comment:
If I try to start the containers _without_ copying, I get an error:
```
ERROR: Couldn't find env file: /Users/jschladen/repos/lemur-docker/.pgsql.env
```

So this is now a core step. I'd move these steps into the block above to make that clear (and maybe add a note in the "Configuration" section that you can edit them if needed).

Reply:
Yes, it is expected behavior. I can create Makefile  with logic:
```bash
#  if env file doesn't exists just warn to console and copy it before starting containers.
```
I can't add note because my English very poor, sorry for that.

Reply:
No problem at all. We can leave it as-is for now (I'll add a note later), but that behavior of copying the first time seems reasonable too if you want to add it.